### PR TITLE
Group minio and minio-client into same directory

### PR DESCRIPTION
**Pull request by tcnghia**
This follows the convention for Kyverno and Tekton, which allows us to test both of these images together.

### CODE REVIEW

File: images/minio/main.tf [hunk 1]
```diff
@@ -8,47 +8,59 @@ variable "target_repository" {
...
 module "latest" {
-  source = "../../tflib/publisher"
+  for_each = local.packages
+  source   = "../../tflib/publisher"
 
   name = basename(path.module)
 
-  target_repository = var.target_repository
-  config            = file("${path.module}/configs/latest.apko.yaml")
+  target_repository = "${var.target_repository}${trimprefix(each.key, "minio")}"
```
Comment (mattmoor) [inline images/minio/main.tf:24]: look at what kyverno does here.

File: images/minio/main.tf [hunk 2]
```diff
@@ -8,47 +8,59 @@ variable "target_repository" {
...
+  package  = each.value
+  config   = module.latest[each.key].config
 }
 
 module "test-latest" {
-  source = "./tests"
-  digest = module.latest.image_ref
+  for_each = local.packages
+  source   = "./tests"
+  digest   = module.latest[each.key].image_ref
```
Comment (mattmoor) [inline images/minio/main.tf:53]: We should make this pass all of the constituent images as `digests = { ... }` and then we can run the current `--version` test on each, which will make the follow up to test them against each other much more contained.

Comment (tcnghia) [reply images/minio/main.tf]: Done

File: images/minio/main.tf (hunk 1 above)
Comment (tcnghia) [reply images/minio/main.tf]: Done

File: images/minio/tests/main.tf [hunk 3]
```diff
@@ -4,11 +4,13 @@ terraform {
   }
 }
 
-variable "digest" {
-  description = "The image digest to run tests over."
+variable "digests" {
+  description = "The image digests to run tests over."
```
Comment (mattmoor) [inline images/minio/tests/main.tf:8]: generally this enumerates the keys in a type, check one of the example images.

Comment (tcnghia) [reply images/minio/tests/main.tf:8]: Done.

**Review by mattmoor — APPROVED** [review]

**Pull request merged**